src/components/Footer.tsx

```diff
@@ -0,0 +1,36 @@
+'use client';
+import Link from 'next/link';
+import Image from 'next/image';
+import MouseEvent from 'react';
```

Comment:
[Required] MouseEvent 是從 'react' 解構出來的
```suggestion
import { MouseEvent } from 'react';
```

Reply:
✅ 已在 2024/6/01 19:38 的 commit : 5880082  修改完畢
      （引入 MouseEventHandler時，一併和 MouseEvent 以此種 import方式）